platform-hub-api/app/models/kubernetes_group.rb

```diff
@@ -74,4 +76,12 @@ def handle_destroy
     KubernetesToken.update_all_group_removal self.name
   end
 
+  def cluster_names_exists
+    Array(self.restricted_to_clusters).each do |name|
```

Comment:
Do you need to cast to Array specifically?

Reply:
I suppose it's not _necessary_; it's more to be defensive, in case it's `nil`.